command/command.go

```diff
@@ -15,17 +15,16 @@ import (
 var (
 	// Could be modified in tests.
 	maxGUBatchSize       = 500
-	maxClientObjectQuota = 85000
+	maxClientObjectQuota = 60000
```

Comment:
Does this downgrade in size need to be done in two steps?
ex: roll the change out first (since people already have this object count), and then roll the quota change out next?

If we calculate the true object count (prune expired) before any operations, we should be fine with this in one step. I just am not familiar with when that logic runs

Reply:
>If we calculate the true object count (prune the expired ones) before any operations, we should be fine with this in one step.

yep, exactly. the next time the client attempts a sync, we'll realize that the stored item count is outdated and query the DB for accurate item counts. so one step should suffice.